styles/base.css

```diff
@@ -131,34 +151,74 @@ h6 {
-}
-
-.secondary-btn {
-    text-decoration: none;
-    border: 3px solid var(--green);
-    color: var(--white);
-    padding: 1rem;
-    margin-right: 15px;
-    margin-bottom: 10px;
-}
-
-.secondary-btn:hover {
-    opacity: 70%;
-    color: var(--brand-primary-black);
-    background-color: var(--green);
+:root {
+    --button-primary-default: var(--brand-primary-green);
+    --button-primary-hover: #A6D15F;
+    --button-primary-active: #698A31;
+    --button-primary-focus: var(--brand-primary-green);
+    --button-primary-disabled: #BBB;
+    --button-outline-hover-bg: #2F440C;
+    --button-outline-hover-border: var(--brand-primary-green);
+    --button-outline-active-bg: #213008;
+    --button-outline-disabled-bg: transparent;
+}
+
+.btn {
+    --bs-btn-padding-x: calc(1rem - 1px);
+    --bs-btn-padding-y: calc(1rem - 1px);
+    --bs-btn-font-family: var(--bs-font-monospace);
+    --bs-btn-font-size: 1rem;
+    --bs-btn-font-weight: 400;
+    --bs-btn-line-height: 1;
+    --bs-btn-border-width: calc(1rem / 16);
+    --bs-btn-border-radius: 0;
+    --bs-btn-box-shadow: none;
+    --bs-btn-disabled-opacity: 1;
+    --bs-btn-focus-box-shadow: var(--focus-box-shadow);
+}
+
+.btn-primary {
+    --bs-btn-color: var(--brand-primary-black);
+    --bs-btn-bg: var(--button-primary-default);
+    --bs-btn-border-color: var(--button-primary-default);
+    --bs-btn-hover-color: var(--brand-primary-black);
+    --bs-btn-hover-bg: var(--button-primary-hover);
+    --bs-btn-hover-border-color: var(--button-primary-hover);
+    --bs-btn-focus-shadow-rgb: 49,132,253;
+    --bs-btn-active-color: var(--brand-primary-black);
+    --bs-btn-active-bg: var(--button-primary-active);
+    --bs-btn-active-border-color: var(--button-primary-active);
+    --bs-btn-active-shadow: inset 0 calc(3rem / 16) calc(5rem / 16) rgba(0, 0, 0, 0.125);
+    --bs-btn-disabled-color: var(--brand-primary-black);
+    --bs-btn-disabled-bg: var(--button-primary-disabled);
+    --bs-btn-disabled-border-color: var(--button-primary-disabled);
+}
+
+.btn-outline-primary {
+    --bs-btn-color: var(--white);
+    --bs-btn-bg: transparent;
+    --bs-btn-border-color: var(--button-primary-default);
+    --bs-btn-hover-color: var(--white);
+    --bs-btn-hover-bg: var(--button-outline-hover-bg);
+    --bs-btn-hover-border-color: var(--button-outline-hover-border);
+    --bs-btn-active-color: var(--white);
+    --bs-btn-active-bg:var(--button-outline-active-bg);
+    --bs-btn-active-border-color: var(--button-outline-hover-border);
+    --bs-btn-active-shadow: none;
+    --bs-btn-disabled-color: var(--button-primary-disabled);
+    --bs-btn-disabled-bg: var(--button-outline-disabled-bg);
+    --bs-btn-disabled-border-color: var(--button-primary-disabled);
+    --bs-gradient: none;
+}
+
+.btn-primary:focus,
+.btn-primary:focus-visible,
+.btn-outline-primary:focus,
+.btn-outline-primary:focus-visible {
+    background-color: var(--bs-btn-bg);
+    border-color: var(--bs-btn-border-color);
```

Comment:
This is a really, really neat nesting CSS Variable feature.

The underlying Figma logic calls for: For `btn-primary` and `btn-outline-primary`, the background color on focus should be the same as the default background color. Same with the border color. Bootstrap code has these values set to the _hover_ colors instead - which is brighter - instead of the default color.

In the design, the background color for Primary is green, and the background color for Outline is black. 

In the code, this is how those values are set:

```css
.btn-primary {
    --bs-btn-bg: var(--button-primary-default);
    --bs-btn-border-color: var(--button-primary-default);
}

.btn-outline-primary {
    --bs-btn-bg: transparent;
    --bs-btn-border-color: var(--button-primary-default);
```

In this code below, I'm able to set the background-color to be the `var(--bs-btn-bg)` _in each class's context_. Even though `var(--bs-btn-bg)` is set to _different_ values for `.btn-primary` and `.btn-outline-primary`, the CSS figures out what value is being scoped. Saves time and lines of code.

```
.btn-primary:focus,
.btn-primary:focus-visible,
.btn-outline-primary:focus,
.btn-outline-primary:focus-visible {
    background-color: var(--bs-btn-bg);
    border-color: var(--bs-btn-border-color);
}
```